### PR TITLE
'Session closed' email: mention no action required #7144

### LINKED ISSUE #7144: 'Session closed' email: mention no action required
Current:
TEAMMATES sends an email to student at the time a session closes, and also, 24 hours before closing

Suggestion:
In the email body, mention something like 'no action required if already submitted'

Justification:
Some students write back to say 'no idea what to do for this email, I already submitted'

### PR DESCRIPTION
**Pull request by shrut1996**
Fixes #7144 

I have made the message bold so that the students pay attention to it and not just skim through the email. Is this fine or should I modify this?

### CODE REVIEW

**Comment by damithc** [comment]
> I have made the message bold so that the students pay attention to it and not just skim through the email. Is this fine or should I modify this?

It's fine. Remember to do this for both emails: the one sent 24 hours before closing and the one sent at close

**Comment by shrut1996** [comment]
@damithc Sure, I was just confirming whether the bold message was alright :)

**Comment by HirdayGupta** [comment]
The existing `Session Closing` email already has a line that says `You may ignore this email if you have already submitted feedback.`. Maybe just make that bold instead?

**Comment by damithc** [comment]
> The existing Session Closing email already has a line that says You may ignore this email if you have already submitted feedback.. Maybe just make that bold instead of adding another line that says the same thing?

Sure.

File: src/main/resources/userEmailTemplate-feedbackSessionClosing.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <br><strong>To submit feedback for the above session, please go to this Web address: </strong>
     <a href="${submitUrl}">${submitUrl}</a>
     <br>*The above link is unique to you. Please do not share it with others.
-    <br>You may ignore this email if you have already submitted feedback.
+    <br><strong>You may ignore this email if you have already submitted feedback.</strong>
```
Comment (damithc) [inline src/main/resources/userEmailTemplate-feedbackSessionClosing.html]: Shall we use the same phrasing in both emails? `No action is required if you have already submitted responses for this session.`

File: src/main/resources/userEmailTemplate-feedbackSessionClosed.html [hunk 2]
```diff
@@ -3,6 +3,7 @@
 <p>
     The following session is now closed. You can still view your submission by going to the link sent earlier,
     but you will not be able to edit existing responses or submit new responses.
+    <strong>No action is required if you have already submitted.</strong>
```
Comment (damithc) [inline src/main/resources/userEmailTemplate-feedbackSessionClosed.html]: Perhaps put this in line 13 instead? It goes together with instructions on how to request extensions.

File: src/main/resources/userEmailTemplate-feedbackSessionClosing.html (hunk 1 above)
Comment (shrut1996) [inline src/main/resources/userEmailTemplate-feedbackSessionClosing.html]: @damithc Yes that'll be good and should I leave this here or put it in line 13 aswell?

Comment (damithc) [inline src/main/resources/userEmailTemplate-feedbackSessionClosing.html]: Yes, can be pushed down to line 16.

Comment (shrut1996) [inline src/main/resources/userEmailTemplate-feedbackSessionClosing.html]: Ah yes line 16. My bad :)

**Comment by shrut1996** [comment]
Ready for review @damithc

**Pull request merged**